include/elliptics/packet.h

```diff
@@ -1244,6 +1244,7 @@ struct dnet_server_send_request {
 	int		id_num;
 	int		group_num;
 	uint64_t	iflags;
+	uint64_t	reserved[10];
```

Comment:
It is needed for future use. I suppose that 80 isn't too much, so I suggested it. Do you have another suggestion?